py/embedded-server/deephaven_server/start_jvm.py

```diff
@@ -39,7 +51,6 @@ def start_jvm(
         jvm_args = DEFAULT_JVM_ARGS,
         jvm_properties = DEFAULT_JVM_PROPERTIES,
         java_home: str = os.environ.get('JAVA_HOME', None),
-        classpath = [],
```

Comment:
I removed it in the sake of simplifying the code. I might even suggest we should limit the parameters even more (ie, not give the user ability to change args, properties, etc). Users who really need it can do all of this logic themselves, I think? I'm not against adding it back in (in a modified form), but I do think it may be functionality that initial users won't necessarily need.

Users can still call start_jvm, yeah?

Reply:
Also, have successfully tested it out w/ `extra_classpath = ["/tmp/my-libs/*.jar"]`